src/views/GameListView.jsx

```diff
@@ -78,21 +80,116 @@ const Filter = styled.div`
       font-size: 32px;
     }
   }
+
+  input:not([type="checkbox"]):not([type="radio"]):not([type="submit"]):not([type="color"]):not([type="button"]):not([type="reset"]) {
+    background-color: #1c1c1c;
+    color: var(--font-color2);
+    max-width: 30rem;
+    display: inline;
+    margin: 2rem;
+
+    &::placeholder {
+      color: var(--font-color2);
+    }
+  }
+`;
+
+const StyledGameFilters = styled.ul`
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+
+  > li {
+    flex: auto;
+    max-width: 25rem;
+    margin: 1rem;
+    padding: 1.5rem 1rem 1rem;
+    text-align: center;
+    border-radius: 0.5rem;
+
+    .filter-title {
+      text-transform: uppercase;
+      color: var(--font-color2);
+      font-size: 1.3rem;
+    }
+
+    &.small-filter {
+      max-width: 20rem;
+    }
+  }
+
+  .language-filter {
+    background: url(${languageSVG}) #1d1d1d no-repeat -35% -25%;
+
+    ul {
+      text-align: left;
+
+      list-style: none;
+      padding: 0;
+      color: var(--font-color2);
+    }
+  }
+
+  .player-filter {
+    background: url(${playerSVG}) #1d1d1d no-repeat -35% -25%;
+  }
+
+  .duration-filter {
+    background: url(${clockSVG}) #1d1d1d no-repeat -40% -40%;
+  }
+`;
+
+const StyledGameResultNumber = styled.p`
+  text-align: center;
 `;
 
 const Content = styled.div`
   background-color: var(--bg-secondary-color);
 `;
 
-const cleanWord = function (word) {
+const cleanWord = (word) => {
   return Diacritics.clean(word).toLowerCase();
 };
 
+const hasIntervalOverlap = (interval1, interval2) => {
+  return interval1[0] <= interval2[1] && interval1[1] >= interval2[0];
+};
+
+const hasRequestedValues = (filterValue, gameSettings) => {
+  if (
+    !filterValue ||
+    filterValue.length == 0 ||
+    !gameSettings ||
+    gameSettings.length == 0
+  ) {
+    return true;
+  }
+  return hasIntervalOverlap(filterValue, gameSettings);
+};
```

Comment:
May be:

```js
const hasRequestedValues = (filterValue, gameSettings) => {
  if (!filterValue?.length || !gameSettings?.length){
      return true;
  }
  return hasIntervalOverlap(filterValue, gameSettings);
};
```

or may be (untested, not sure) but more concise and allow to remove this function and keep only hasIntervalOverlap:

```js
const hasRequestedValues = (interval1=[-Infinity, Infinity], interval2=[-Infinity, Infinity]) => 
  interval1[0] <= interval2[1] && interval1[1] >= interval2[0];
```

Reply:
It could have worked if I didn't receive some intervals as empty arrays from the API.